fetch_threads.py

```
@@ -0,0 +1,33 @@
+import json
+import re
+import shutil
+import subprocess
+import sys
+
+gh = shutil.which('gh')
+if gh is None:
+    sys.exit('gh CLI not found on PATH')
+
+res = subprocess.run([gh, 'api', 'graphql', '-f', 'query=query { repository(owner: "dwojtaszek", name: "zipper") { pullRequest(number: 512) { reviewThreads(first: 100) { nodes { id isResolved comments(first: 5) { nodes { body path line } } } } } } }'], capture_output=True, text=True)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)